Dockerfile

```diff
@@ -2,9 +2,12 @@
 # We just need to use --file to point at it, instead of assuming it is in context.
 
 # Using Conda because pyarrow did not install easily on python base images.
-FROM continuumio/miniconda3
+FROM continuumio/miniconda3:4.7.12
 
 COPY requirements-freeze.txt .
+# For tiff packages
+RUN apt-get update &&\
+      apt-get install -y gcc python3-dev
```

Comment:
```suggestion
# For tiff packages
RUN apt-get update &&\
      apt-get install -y gcc python3-dev
COPY requirements-freeze.txt .
```
Better to keep the pip lines together? I'd think the apt-get list to be more stable than the requirements, so I'd put it first.